blueprints/examples/src/examples/eigen_context.rs

```diff
@@ -0,0 +1,196 @@
+use alloy_primitives::{address, Address, Bytes};
+use gadget_sdk::event_listener::evm::contracts::EvmContractEventListener;
+use gadget_sdk::event_utils::InitializableEventHandler;
+use gadget_sdk::utils::evm::get_provider_http;
+use gadget_sdk::{config::StdGadgetConfiguration, ctx::EigenlayerContext, job, load_abi};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::env;
+use std::ops::Deref;
+
+alloy_sol_types::sol!(
+    #[allow(missing_docs)]
+    #[sol(rpc)]
+    #[derive(Debug, Serialize, Deserialize)]
+    ContextExampleTaskManager,
+    "contracts/out/ContextExampleTaskManager.sol/ContextExampleTaskManager.json"
+);
+
+load_abi!(
+    CONTEXT_EXAMPLE_TASK_MANAGER_ABI_STRING,
+    "contracts/out/ContextExampleTaskManager.sol/ContextExampleTaskManager.json"
+);
+
+#[derive(Clone, EigenlayerContext)]
+pub struct ExampleEigenContext {
+    #[config]
+    pub std_config: StdGadgetConfiguration,
+}
+
+pub async fn constructor(
+    env: StdGadgetConfiguration,
+) -> color_eyre::Result<impl InitializableEventHandler> {
+    let context_example_address = env::var("CONTEXT_EXAMPLE_TASK_MANAGER_ADDRESS")
+        .map(|addr| {
+            addr.parse()
+                .expect("Invalid CONTEXT_EXAMPLE_TASK_MANAGER_ADDRESS")
+        })
+        .unwrap_or_else(|_| address!("0000000000000000000000000000000000000000"));
+
+    let context_example_task_manager =
+        ContextExampleTaskManager::ContextExampleTaskManagerInstance::new(
+            context_example_address,
+            get_provider_http(&env.http_rpc_endpoint),
+        );
+
+    Ok(DemonstrateEigenlayerContextEventHandler {
+        contract: context_example_task_manager,
+        contract_instance: Default::default(),
+        ctx: ExampleEigenContext {
+            std_config: env.clone(),
+        },
+    })
+}
+
+#[job(
+    id = 0,
+    params(event, log),
+    event_listener(
+        listener = EvmContractEventListener<ContextExampleTaskManager::NewTaskCreated>,
+        instance = ContextExampleTaskManager,
+        abi = CONTEXT_EXAMPLE_TASK_MANAGER_ABI_STRING,
+        pre_processor = handle_events,
+    ),
+)]
+pub async fn demonstrate_eigenlayer_context(
```

Comment:
Function names don't need to be super absolutely tied to the example, it's more readable to share in docs that this is a "handle_task" function, not a `demonstrate_eigenlayer_context`, which holds little meaning in our job terminology.